package-lock.json

```diff
@@ -3640,17 +3640,17 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": false,
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Do you know why there are all these changes to the package-lock.json file? I don't think this is related to any of the docker changes, we should probably just discard these changes.